src/screens/DiscoveryScreenV2/fragment/TopicFragment.js

```diff
@@ -382,7 +384,8 @@ TopicFragment.propTypes = {
   setUnfollowedTopic: PropTypes.func,
   fetchData: PropTypes.func,
   searchText: PropTypes.string,
-  withoutRecent: PropTypes.bool
+  withoutRecent: PropTypes.bool,
+  initialTopics: PropTypes.array
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The PropTypes validation for `initialTopics` is correct as it should be an array. However, consider making it more specific by using `PropTypes.arrayOf(PropTypes.shape({}))` to validate the structure of the objects within the array, if you know what the structure should be.
```diff
-  initialTopics: PropTypes.array
+  initialTopics: PropTypes.arrayOf(PropTypes.shape({}))
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->